dlc-manager/src/contract/mod.rs

```diff
@@ -149,8 +160,19 @@ pub struct FailedSignContract {
     pub error_message: String,
 }
 
+/// Information about a contract that is almost closed by a broadcasted, but not confirmed CET.
+#[derive(Clone)]
+pub struct PreClosedContract {
```

Comment:
Since this struct is exactly the same as `ClosedContract` it might be ok to just use the same struct but a different `Contract` enum variant like is already done for e.g. `Signed` and `Confirmed`.

Reply:
Makes sense. I left it as two individual structs with the intention of cutting down on the number of fields in the `ClosedContract` struct as a means of pruning in the future. Would you still prefer one shared over two individual? I agree that if no pruning of information in the `ClosedContract` struct is to occur, they should share.

Reply:
Mmmm fair enough, maybe ok to keep it as is then, would you maybe just add a todo or comment?

Reply:
Sure!